tests/test-cases.js

```diff
@@ -1208,13 +1208,131 @@ var PARSE_TESTS = [
 			data: [{'a': 'c', 'b': 'd'}],
 			errors: []
 		}
+	},
```

Comment:
Probably a test here to verify that using a regexy quote character doesn't break things.